charts/cass-operator/templates/deployment.yaml

```diff
@@ -1,60 +1,45 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Values.deploymentName }}
+  name: {{ include "cass-operator.fullname" . }}
+  labels:
+    {{- include "cass-operator.labels" . | nindent 4 }}
```

Comment:
Now that we would manage the cass-operator, should we include the k8ssandra-labels also?

Reply:
Yeah, we definitely want to include the labels. I think the approach I want to take is to define a common chart that each of our chart pulls in as a dependency. The common chart can define various helper templates, like for labels. See https://github.com/technosophos/common-chart/tree/master/common for an example. Bitnami takes this approach as well https://github.com/bitnami/charts/tree/master/bitnami.